src/panels/lovelace/views/hui-view.ts

```diff
@@ -131,7 +131,12 @@ export class HUIView extends UpdatingElement {
       this._layoutElement!.lovelace = lovelace;
     }
 
-    if (configChanged || hassChanged || editModeChanged) {
+    if (
+      configChanged ||
+      hassChanged ||
+      editModeChanged ||
+      changedProperties.has("_cards" || changedProperties.has("_badges"))
```

Comment:
it doesn't look right

shouldn't it be `changedProperties.has("_cards") || changedProperties.has("_badges")`